app.py

```diff
@@ -35,19 +35,22 @@ def init_app(self):
 
         display_name = self.get_setting("display_name")
 
-        menu_caption = "%s..." % display_name
-        menu_options = {
-            "short_name": "send_for_review",
-            "description": "Send a version for review using Shotgun Create",
-            # dark themed icon for engines that recognize this format
-            "icons": {
-                "dark": {"png": os.path.join(self.disk_location, "icon_256_dark.png")}
-            },
-        }
+        if display_name:
```

Comment:
Could you add a comment why the command is not registered if there is no display name? Why would someone not put one? Should the argument just be required then and people not include the app if they don't intend to use it?

Reply:
This is because the app is used as an API here: https://github.com/shotgunsoftware/tk-nuke/blob/master/hooks/tk-multi-publish2/basic/submit_for_review.py#L128 and we don't want to show the item in the Nuke's menu items.

Reply:
Comment added.